app/src/main/java/com/monstarlab/features/login/LoginFragment.kt

```diff
@@ -21,7 +22,14 @@ class LoginFragment : ComposeFragment() {
     private val viewModel by viewModels<LoginViewModel>()
 
     override val content: @Composable () -> Unit = {
-        LoginScreen(viewModel = viewModel)
+        val state = viewModel.stateFlow.collectAsState().value
+        LoginScreen(state) { event ->
+            when(event) {
+                LoginScreenEvent.LoginButtonPress -> viewModel.login()
+                is LoginScreenEvent.LoginChange -> viewModel.onEmailTextChanged(event.value)
+                is LoginScreenEvent.PasswordChange -> viewModel.onPasswordTextChanged(event.value)
+            }
```

Comment:
I guess its all a matter of preference  :) 
For me it looks like nice and compact this way, you have a state coming in and an event coming out.
Also, in case you have a screen with a bigger amount of user events, and you only have lambda callbacks, this Composable function can become rather big